src/parsers/variable.cr

```diff
@@ -45,7 +45,7 @@ module Mint
       variable(track) || raise error
     end
 
-    def variable(track = true, invalid = [] of String) : Ast::Variable?
+    def variable(track = true, invalid = ["true", "false"]) : Ast::Variable?
```

Comment:
I'd keep the constant for performance reasons.

```suggestion
    def variable(track = true, invalid = INVALID_VARIABLE_NAMES) : Ast::Variable?
```

Reply:
Moved it from the arguments into the method body.